packages/headless/src/features/commerce/facets/facet-set/facet-set-slice.ts

```diff
@@ -540,12 +538,18 @@ function insertNewValue(
   facetRequest.numberOfValues = facetRequest.values.length;
 }
 
-function resetAllFacetValues(state: CommerceFacetSetState) {
+function setAllFacetValuesToIdle(state: CommerceFacetSetState) {
   Object.values(state).forEach((facet) => {
     facet.request.values.forEach((value) => (value.state = 'idle'));
   });
 }
 
+function clearAllFacetValues(state: CommerceFacetSetState) {
```

Comment:
A bit out of scope, but I also noticed that when the context changes, we were only resetting facet value states, not clearing the values entirely.

This implies, for example, that if a user is on PLP A in which a facet with ID X is configured and they navigate to PLP B in which a facet with ID X is also configured, then the facet would be displayed with the values from PLP A in PLP B, which makes no sense.

I believe the correct approach is to clear facet values on context changes, and set facet value states to idle when breadcrumbs are cleared.